### PR TITLE
Issue 1757 covariate table revision

### PR DESCRIPTION
**Pull request by AntonStepanof**
fixes https://github.com/OHDSI/Atlas/issues/1757

### CODE REVIEW

File: js/pages/characterizations/services/conversion/PrevalenceStatConverter.js [hunk 1]
```diff
@@ -57,23 +58,37 @@ define([
                 title: 'Covariate',
                 data: 'covariateName',
                 className: this.classes('col-prev-title'),
+                render: (d, t, r) => {
+                    return utils.extractMeaningfulCovName(d);
+                 },
+            };
+        }
+
+        getExportColumn() {
```
Comment (pavgra) [inline js/pages/characterizations/services/conversion/PrevalenceStatConverter.js]: `export`?

File: js/pages/characterizations/services/conversion/PrevalenceStatConverter.js [hunk 2]
```diff
@@ -57,23 +58,37 @@ define([
...
+            return {
+                title: 'Explore',
+                data: 'explore',
+                className: this.classes('col-prev-title'),
```
Comment (pavgra) [inline js/pages/characterizations/services/conversion/PrevalenceStatConverter.js]: title?

File: js/pages/characterizations/services/conversion/PrevalenceStatConverter.js (hunk 1 above)
Comment (AntonStepanof) [reply js/pages/characterizations/services/conversion/PrevalenceStatConverter.js]: changed to explore

File: js/pages/characterizations/services/conversion/PrevalenceStatConverter.js (hunk 2 above)
Comment (AntonStepanof) [reply js/pages/characterizations/services/conversion/PrevalenceStatConverter.js]: removed this

**Comment by AntonStepanof** [comment]
> Looks not good:
> 
> * The button takes a lot of space while we are trying to save space
> * No padding between the caption and icon in the button
>   image
> * Cursor is wrong
>   image

updated button, paddings and cursor

**Pull request merged**